docs/helm.md

```diff
@@ -61,38 +61,44 @@ Kubernetes: `>=1.20`
 | sidecars.attacherImage.httpEndpointPort | string | `"8090"` | Port of the http endpoint |
-| sidecars.attacherImage.leaderElection | object | `{}` | Customize leaderElection, you can specify `leaseDuration`, `renewDeadline` and/or `retryPeriod`. Each value must be in an acceptable time.ParseDuration format.(Ref: https://pkg.go.dev/flag#Duration) |
+| sidecars.attacherImage.leaderElection | object | `{}` |  |
 | sidecars.attacherImage.repository | string | `"registry.k8s.io/sig-storage/csi-attacher"` |  |
-| sidecars.attacherImage.tag | string | `"v3.3.0"` |  |
+| sidecars.attacherImage.tag | string | `"v4.7.0"` |  |
 | sidecars.livenessProbeImage.port | string | `"9808"` | Port of the liveness of the main container |
 | sidecars.livenessProbeImage.repository | string | `"registry.k8s.io/sig-storage/livenessprobe"` |  |
-| sidecars.livenessProbeImage.tag | string | `"v2.5.0"` |  |
+| sidecars.livenessProbeImage.tag | string | `"v2.14.0"` |  |
 | sidecars.nodeDriverRegistrarImage.enableHttpEndpoint | bool | `false` | Enable http endpoint to get metrics of the container |
 | sidecars.nodeDriverRegistrarImage.enableLivenessProbe | bool | `false` | Enable liveness probe for the container |
 | sidecars.nodeDriverRegistrarImage.httpEndpointPort | string | `"8093"` | Port of the http endpoint |
 | sidecars.nodeDriverRegistrarImage.repository | string | `"registry.k8s.io/sig-storage/csi-node-driver-registrar"` |  |
-| sidecars.nodeDriverRegistrarImage.tag | string | `"v2.3.0"` |  |
+| sidecars.nodeDriverRegistrarImage.tag | string | `"v2.12.0"` |  |
+| sidecars.provisionerImage.additionalArgs | list | `[]` | Customize leaderElection, you can specify `leaseDuration`, `renewDeadline` and/or `retryPeriod`. Each value must be in an acceptable time.ParseDuration format.(Ref: https://pkg.go.dev/flag#Duration) |
+| sidecars.provisionerImage.additionalClusterRoleRules | string | `nil` |  |
 | sidecars.provisionerImage.enableHttpEndpoint | bool | `false` | Enable http endpoint to get metrics of the container |
 | sidecars.provisionerImage.enableLivenessProbe | bool | `false` | Enable liveness probe for the container |
-| sidecars.provisionerImage.extraArgs.--feature-gates | string | `"NodeSwap=true"` |  |
-| sidecars.provisionerImage.extraArgs.--feature-gates | string | `"CrossNamespaceVolumeDataSource=true"` |  |
 | sidecars.provisionerImage.httpEndpointPort | string | `"8089"` | Port of the http endpoint |
-| sidecars.provisionerImage.leaderElection | object | `{}` | Customize leaderElection, you can specify `leaseDuration`, `renewDeadline` and/or `retryPeriod`. Each value must be in an acceptable time.ParseDuration format.(Ref: https://pkg.go.dev/flag#Duration) |
+| sidecars.provisionerImage.leaderElection | object | `{}` |  |
 | sidecars.provisionerImage.repository | string | `"registry.k8s.io/sig-storage/csi-provisioner"` |  |
-| sidecars.provisionerImage.tag | string | `"v3.0.0"` |  |
+| sidecars.provisionerImage.tag | string | `"v5.1.0"` |  |
```

Comment:
And theses:
- [volume_mode](https://github.com/kubernetes-csi/external-provisioner/blob/master/CHANGELOG/CHANGELOG-4.0.md)
- [topology](https://github.com/kubernetes-csi/external-provisioner/blob/master/CHANGELOG/CHANGELOG-5.0.md)
- [permissions csi-provisioner](https://github.com/kubernetes-csi/external-provisioner/blob/master/CHANGELOG/CHANGELOG-5.0.md)
- [module path v5](https://github.com/kubernetes-csi/external-provisioner/blob/master/CHANGELOG/CHANGELOG-5.1.md)

Reply:
for this [volume_mode](https://github.com/kubernetes-csi/external-provisioner/blob/master/CHANGELOG/CHANGELOG-4.0.md), Since we haven’t explicitly enabled --prevent-volume-mode-conversion=true in our Helm chart, it means the CSI driver will maintain its current behavior

Reply:
I will only upgrade the provisioner until v4.0.0, for the rest of changes, it will be in a seperate PR as it needs some changes.